test/old_tests/UnitTests/Errors.cpp

```diff
@@ -200,6 +200,8 @@ TEST_CASE("Errors")
     SetLastError(ERROR_CANCELLED);
     REQUIRE_THROWS_AS(check_bool(static_cast<BOOL>(false)), hresult_canceled);
 
+    REQUIRE_NOTHROW(check_bool(true) == true);
```

Comment:
`REQUIRE_NOTHROW` just asserts that the expression does not throw an exception. What you're trying to do is check that the expression is true. For that you should use the `REQUIRE` test macro.
```suggestion
    REQUIRE(check_bool(true) == true);
```